src/inverted-index.js

```diff
@@ -0,0 +1,134 @@
+/*  eslint-disable no-unused-vars*/
+/* jshint esnext: true */
+/**
+ * InvertedIndex class with constructor
+ * @class
+ */
+class InvertedIndex {
+  /**
+   * class constructor
+   * @constructor
+   */
+  constructor() {
+    this.file = {};
+    this.index = {};
+    this.docNumber = {};
+  }
+
+  /**
+   *
+   *
+   * @param {any} fileContent
+   * @param {any} fileName
+   * @returns {string} returns createIndex
+   * @memberOf InvertedIndex
+   */
+  createIndex(fileContent, fileName) {
+    this.docCount = [];
+    fileContent.forEach((bookContent, docIndex) => {
+      this.docCount.push(parseInt(docIndex, 10));
+      this.combinedWord = (`${bookContent.title} ${bookContent.text}`).toLowerCase();
+      const token = InvertedIndex.tokenize(this.combinedWord);
+      const remove = new Set(token);
+      const getUnique = Array.from(remove);
+      this.mapWords(getUnique, docIndex);
+    });
+    this.file[fileName] = this.index;
+    this.docNumber[fileName] = this.docCount;
+    this.index = {};
+  }
+  /**
+   *
+   *
+   * @static
+   * @param {any} words
+   * @returns {string} returns tokenize words
+   *
+   * @memberOf InvertedIndex
+   */
+  static tokenize(words) {
+    const tokenizedWords = words.match(/\w+/g);
+    return tokenizedWords;
```

Comment:
**Nitpick - Code Quality**

Variable assignment is not necessary here. You can just return the expression.
```javascript
return words.match(/\w+/g);
```